src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background: #e5e5e5;
+}
+
+.header {
+  padding: 0, 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
```

Comment:
You have set padding for header wrong (see above)
so you don't need to set margin for logo and nav separately. just fix padding for entire header